slam-libraries/viam-cartographer/src/io/submap_painter.cc

```diff
@@ -113,5 +113,27 @@ cartographer::io::PaintSubmapSlicesResult PaintSubmapSlices(
                                                      origin);
 }
 
+// Draw a position on the current map
+void DrawPositionOnSurface(
```

Comment:
[nit] Use the word "pose" instead of "position" to be consistent with the naming so far. `DrawPoseOnSurface` instead of `DrawPositionOnSurface`

Reply:
Only thing was keeping the name consistent with the GRPC position endpoint since these functions all relate to that(using the same protected variable) but I am fine with changing to pose